client/src/composables/useFlights.js

```diff
@@ -21,18 +26,30 @@ export default () => {
 
   // Actions
 
-  const fetchFlights = async (params, queries) => {
+  const fetchFlights = async ({
+    params,
+    queries,
+    limit = DEFAULT_LIMIT,
+    offset = 0,
+  } = {}) => {
     if (params) paramsCache.value = params;
     if (queries) filterOptionsCache.value = queries;
+    if (offset < 0) offset = 0;
+
     try {
       const res = await ApiService.getFlights({
         ...paramsCache.value,
         sortCol: sortOptionsCache.value?.sortCol,
         sortOrder: sortOptionsCache.value?.sortOrder,
         ...filterOptionsCache.value,
+        limit,
+        offset,
       });
       if (res.status != 200) throw res.statusText;
-      flights.value = res.data;
+      flights.value = res.data.rows;
```

Comment:
wo kommt dieses rows eigentlich her?

Reply:
Im Backend habe ich findAll auf auf findAllAndCount umgestellt. D.h. ich bekomme alle Werte (die stehen in rows) und die Gesamtzahl aller Treffer (unabhängig ob ich "limit" genutzt habe). Über diesen Mechanismus kann ich mich dann in der Pagination bewegen.